internal/runners/run/run.go

```diff
@@ -83,7 +83,7 @@ func run(out output.Outputer, subs subshell.SubShell, proj *project.Project, nam
 	if !lang.Recognized() {
 		warning := locale.Tl(
 			"run_warn_deprecated_script_without_language",
-			"[YELLOW]DEPRECATION WARNING: Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
+			"[NOTICE][HIGHLIGHT]DEPRECATION WARNING:[/RESET][NOTICE] Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
```

Comment:
```suggestion
			"[NOTICE][HIGHLIGHT]DEPRECATION WARNING:[/RESET][NOTICE] Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
```

Is this covered in the UX mocks? ie. using HIGHLIGHT here at all.

Reply:
You are probably right. It just says

> great for highlights and actionable content

But it is not really actionable.  So, I think I will remove the HIGHLIGHT.